configs.py

```diff
@@ -0,0 +1,45 @@
+# HOST = 'https://dev.api.it-sl.ru'
```

Comment:
This commented out string should be removed 😄

Reply:
Resolve